bazel/cc_toolchains/clang_toolchain.BUILD

```diff
@@ -85,3 +86,23 @@ cc_toolchain_config(
     name = "local-darwin-arm64",
     target_cpu = "darwin_arm64",
 )
+
+cc_toolchain(
+    name = "cc-compiler-x64-windows",
+    all_files = ":empty",
+    ar_files = ":empty",
+    as_files = ":empty",
+    compiler_files = ":empty",
+    dwp_files = ":empty",
+    linker_files = ":empty",
+    objcopy_files = ":empty",
+    strip_files = ":empty",
+    supports_param_files = 1,
+    toolchain_config = ":local-x64-windows",
+    toolchain_identifier = "local-x64-windows",
+)
+
+cc_toolchain_config(
+    name = "local-x64-windows",
+    target_cpu = "x64_windows",
```

Comment:
As the cc_toolchain error has my attention: is there a reason this is `x64_windows` instead of `windows_x64`, echoing darwin?

Reply:
I am following this [Bazel document](https://docs.bazel.build/versions/main/windows.html) where the cpu is referenced as x64_windows. I don't know if this is a Bazel/system specific thing or if I can change it to be similar to the darwin one. I will investigate.